header.html

```diff
@@ -0,0 +1,82 @@
+document.write(`

```

Comment:
Hi David! Sorry for the confusion. You don't need to create a new header.html file. To add the FAQ to the navbar, you would edit the existing header.js file.